Dockerfile_dependencies

```diff
@@ -16,4 +16,4 @@ RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
         mv conanprofile.docker conanprofile; \
     fi
 
-RUN conan install . --build=missing --profile ./conanprofile --profile:build ./conanprofile --output-folder=build
+RUN conan install . --build=missing --profile ./conanprofile --profile:build ./conanprofile --output-folder=cmake
```

Comment:
(Not necessarily for this PR): I wonder why we don't simply use the `build_with_conan` script here? Actually there is no need to replicate this command in several places. And installing Python in the build stage of the Dockerfile would be no big deal.